dialect/sql/schema/postgres.go

```diff
@@ -403,7 +403,12 @@ func (d *Postgres) writeDefault(b *sql.ColumnBuilder, c *Column) {
 			attr = fmt.Sprintf("'%s'", strings.ReplaceAll(v, "'", "''"))
 		}
 	}
-	b.Attr("DEFAULT " + attr)
+
+	var sb strings.Builder
+	sb.WriteString(clause)
+	sb.WriteString(attr)
+
+	b.Attr(sb.String())
```

Comment:
```suggestion
	b.Attr(caluse + " " + attr)
```

string builder is not needed here.